Tchap/Modules/Home/HomeCoordinator.swift

```diff
@@ -211,6 +211,15 @@ final class HomeCoordinator: NSObject, HomeCoordinatorType {
         }
     }
     
+    private func showFavourites(animated: Bool) {
+        let favouritesCoordinator = FavouritesCoordinator(session: self.session)
```

Comment:
My suggestion at least is to use directly `FavouriteMessagesCoordinator` here , but perhaps i'm wrong

Reply:
You right. Now, I use `FavouriteMessagesCoordinator` directly.